pages/blog/_slug.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="bg-light-elevatedSurface dark:bg-dark-elevatedSurface">
```

Comment:
This section should have shadow (you can notice it right about "NuxtJS Newsletter" well)